### PR TITLE
feat: Add k6 to TRIAL

### PR DESCRIPTION
**Pull request by SnigL**

### CODE REVIEW

File: radar/qa/k6.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    ring: TRIAL
+description: |
+  [k6](https://k6.io/) is a modern open-source load testing tool written in Go and executes scripts written in
+  Javascript.
+rationale: |
+  By using k6 we can easily identify performance issues early in development by "shifting left" and integrating it with
+  our CI pipeline, making it an ideal tool for our new services and products.
+
+  Checks and thresholds can be added to the scripts as pass/fail criterias along with outputing the result to a number
+  of products such as Kafka, Datadog and InfluxDB to mention a few. They also have a SaaS product running in the Cloud.
```
Comment (Tasseri) [inline radar/qa/k6.yaml:14]: This is a description and not rationale

Comment (SnigL) [reply radar/qa/k6.yaml]: Re-wrote the description and the rationale

**Comment by Tasseri** [comment]
@SnigL Looks good, you need to rebase and fix the inital commit from "feature: .." to "feat: .."

**Comment by SnigL** [comment]
> @SnigL Looks good, you need to rebase and fix the inital commit from "feature: .." to "feat: .."

Done and done

**Pull request merged**